olive/passes/pytorch/lora.py

```diff
@@ -671,7 +671,9 @@ def train_and_save_new_model(
         # save adapter weights
         adapter_path = Path(output_model_path) / "adapter"
         adapter_path.mkdir(parents=True, exist_ok=True)
-        model.save_pretrained(adapter_path)
+        # don't save embedding layers since only adapter weights are trained
+        # if we don't provide as False, it defaults to "auto" which checks if the vocab size changed
+        model.save_pretrained(adapter_path, save_embedding_layers=False)
```

Comment:
does the save_embedding_layers need to be configurable by exposing it in the pass config?

Reply:
it doesn't. We only train the adapters. There are more details under the `smart_tokenizer_...` method.